src/NUnitConsole/nunit3-console/CommandLineOptions.cs

```diff
@@ -402,7 +400,7 @@ private void ResolveOutputSpecification(string value, IList<OutputSpecification>
 
             if (spec.Transform != null)
             {
-                var transformPath = Path.Combine(WorkDirectory, spec.Transform);
+                var transformPath = Path.Combine(Environment.CurrentDirectory, spec.Transform);
```

Comment:
@CharliePoole I was actually more worried about the opposite situation that we would instantiate `CommandLineOptions` from several locations and then then `DEFAULT_WORK_DIRECTORY` would be fixed to something that was not correct anymore. So I only checked that `CommandLineOptions` was only constructed once. I will reintroduce the static. 

When looking more into this I have some questions regarding `WorkDirectory`. In https://github.com/nunit/nunit-console/blob/0500ce8c3216157d320b6072ae2b57544e5d9362/src/NUnitConsole/nunit3-console/ConsoleRunner.cs#L77 and https://github.com/nunit/nunit-console/blob/0500ce8c3216157d320b6072ae2b57544e5d9362/src/NUnitConsole/nunit3-console/ConsoleRunner.cs#L414 we check whether `WorkDirectory` is null and then use `Environment.CurrentDirectory` as a substitute. I cannot see that `WorkDirectory` can be null so these "guards" make little sense, but the last guard was introduced a year ago, so perhaps I've overlooked something?